gobblin-audit/src/main/java/gobblin/audit/policy/column/ColumnProjectionPolicy.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.audit.policy.column;
+
+import java.util.List;
+
+import org.apache.avro.generic.GenericRecord;
+
+
+/**
+ * An interface that projects certain columns/fields of an input {@link GenericRecord} to generate a new {@link GenericRecord} that can be audited.
```

Comment:
Never mind, I noticed that of the AbstractColumnProjectionPolicy javadocs. I suggest moving it here. The naming is part of the contract.